w3-space.md

```diff
@@ -0,0 +1,126 @@
+# Space
+
+![status:reliable](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square)
+
+## Editors
+
+- [Vasco Santos], [Protocol Labs]
+
+## Authors
+
+- [Vasco Santos], [Protocol Labs]
+
+## Abstract
+
+A Space can be defined as a namespace for stored content. It is created locally, offline, and associated with a cryptographic key pair (identified by the did:key of the public key). Afterwards, a space MAY be registered with a web3.storage [account](./w3-account.md) and [providers](./w3-provider.md) MAY be added to the space.
+
+- [Capabilities](#capabilities)
+  - [`space/info`](#spaceinfo)
+  - [`space/allocate`](#spaceallocate)
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Capabilities
+
+### `space/info`
+
+Get information about a given space, such as registered providers.
+
+> `did:key:zAliceAgent` invokes `space/info` capability provided by `did:web:web3.storage`
+
+```json
+{
+  "iss": "did:key:zAliceAgent",
+  "aud": "did:web:web3.storage",
+  "att": [
+    {
+      "with": "did:key:zAlice",
+      "can": "space/info"
+    }
+  ],
+  "prf": [],
+  "sig": "..."
+}
+```
+
+#### Space Info Failure
+
+```json
+{
+  "ran": "bafy...spaceInfo",
+  "out": {
+    "error": {
+      "name": "SpaceUnknown"
+    }
+  }
+}
+```
+
+#### Space Info Success
+
+```json
+{
+  "ran": "bafy...spaceInfo",
+  "out": {
+    "ok": {
+      "did": "did:key:zAlice",
+      "providers": [
+        "did:web:free.web3.storage"
+      ]
+    }
+  }
+}
+```
+
+### `space/allocate`
```

Comment:
Do we want to document this? AFAIK it is not used by our client/server as an invoked capability...although the handler _is_ used by `store/add` handler.

However it doesn't do anything related to allocating space - it rate limits and checks for storage provider.

Reply:
it is used by `store/add` yes. I think it should be documented, as `store/add` may happen in another place in the future. 

> However it doesn't do anything related to allocating space - it rate limits and checks for storage provider.

I think because it is not implemented? But that is the goal here? We can make that clear in the spec if that would be the desire.

Maybe @travis can help out with decision here

Reply:
> it is used by `store/add` yes. I think it should be documented, as `store/add` may happen in another place in the future.

Right but I think that should be in `w3-store.md` and should say that `store/add` MAY rate limit and SHOULD ensure the space is provisioned properly.

> I think because it is not implemented? But that is the goal here? We can make that clear in the spec if that would be the desire.

I would just remove it - it is not invoked and does nothing to do with allocating space. The handler function should instead be a lib/utility function.

Reply:
ok, I removed until we decide what to do